packages/querygrid/src/models.ts

```diff
@@ -732,6 +735,61 @@ export class EditorModel extends Record({
             this.selectedRowIdx === rowIdx
         );
     }
+
+    static getEditorDataFromQueryValueMap(valueMap: any) : List<any> | any {
```

Comment:
Is List<any> useful if any is also specified?

Reply:
Perhaps not technically, but I like knowing that it can be either a list or an object.